develop/develop-images/build_enhancements.md

```diff
@@ -215,14 +220,16 @@ RUN mkdir -p -m 0600 ~/.ssh && ssh-keyscan github.com >> ~/.ssh/known_hosts
 RUN --mount=type=ssh git clone git@github.com:myorg/myproject.git myproject
 ```
 
-Once the `Dockerfile` is created, use the `--ssh` option for connectivity with
-the SSH agent.
+The image could be built as follows:
 
 ```bash
 $ docker build --ssh default .
 ```
 
-You may need to run `ssh-add` to add private key identities to the authentication agent first for this to work.
+As with `--mount=type=secret`, you can specify an `id` to differentiate sockets.
```

Comment:
you can specify an `id` if you want to use multiple sockets per build and want to differentiate them.